docs/data-model.md

```diff
@@ -12,6 +12,7 @@
   - [Users](#users)
   - [Tags](#tags)
   - [Environments](#environments)
+    - [LhcFills](#lhcfills)
```

Comment:
I am not sure the indentation is correct for this one, and you may add a space after this line

Reply:
I missed this, thank you for spotting it. For some reason my on-save-formater insists on adding that tab. 
Removed.